backend/pkg/api/events.go

```diff
@@ -164,8 +164,7 @@ func (api *API) triggerEventConsequences(instanceID, appID, groupID, lastUpdateV
 		return err
 	}
 
-	// TODO: should we also consider ResultSuccess in the next check? Flatcar ~ generic conflicts?
-	if etype == EventUpdateComplete && result == ResultSuccessReboot {
+	if etype == EventUpdateComplete && (result == ResultSuccessReboot || result == ResultSuccess) {
```

Comment:
```suggestion
	if etype == EventUpdateComplete && (result == ResultSuccessReboot || result == ResultSuccess) {
```
I would make it a property of the application whether `ResultSuccessReboot` or `ResultSuccess` is expected here.
Instead of adding a DB property you can also add a check here against the hardcoded Flatcar application ID.

Reply:
@pothos Did you mean to add a code suggestion?

So AFAIU, you're saying "keep this change you added but only if the app is not FCL".

Reply:
Thanks, looks good now.

Yes, I didn't want to add a suggestion but just a comment on that line.